File-Formats/DocIO/Create-Word-document-in-MAUI.md

```diff
@@ -305,7 +305,7 @@ You can download a complete working sample from [GitHub](https://github.com/Sync
 
 By executing the program, you will get the **Word document** as follows.
 
-![MAUI output Word document](MAUI_Images/GettingStartedOutput.jpg)
+![MAUI output Word document](maui_images/gettingstarted-output-in-maui-word.jpg)
```

Comment:
image name should be meaningful. Read the page content and provide proper suitable name for image.
Change like as below,
generated-word-document-in-maui

Reply:
changed